android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/LocationModule.java

```diff
@@ -154,6 +174,7 @@ private void stopWatching() {
     }
 
     SmartLocation.with(mScopedContext).location().stop();
+    SmartLocation.with(mScopedContext).geocoding().stop();
```

Comment:
@aalices @jesseruder Unfortunately this seems to break in my environment. Recommend wrapping this line with `Geocoder.isPresent()` or some equivalent?

![image](https://user-images.githubusercontent.com/649892/28486838-af46afe0-6e3b-11e7-86e2-57159388218f.png)